### PR TITLE
Implement two Unglued cards

### PR DESCRIPTION
**Pull request by ketsuban**
I decided to get a bit more adventurous when implementing missing cards, adding new costs and things.

### CODE REVIEW

File: Mage.Sets/src/mage/cards/m/MissDemeanour.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        return new MissDemeanourEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Player controller = game.getPlayer(source.getControllerId());
+        Permanent sourceObject = (Permanent) source.getSourceObjectIfItStillExists(game);
+        if (sourceObject != null) {
+            if (!controller.chooseUse(outcome, "Compliment " + game.getPlayer(game.getActivePlayerId()).getName() + " on their game play?", source, game)) {
+                sourceObject.sacrifice(source.getSourceId(), game);
```
Comment (JayDi85) [inline Mage.Sets/src/mage/cards/m/MissDemeanour.java]: Missing `game.informPlayers(controller.getLogName() + ": " + some compliment to active player);` :-)

File: Mage.Sets/src/mage/cards/i/InfernalSpawnOfEvil.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        super(card);
+    }
+
+    @Override
+    public InfernalSpawnOfEvil copy() {
+        return new InfernalSpawnOfEvil(this);
+    }
+}
+
+class SayCost extends CostImpl {
```
Comment (JayDi85) [inline Mage.Sets/src/mage/cards/i/InfernalSpawnOfEvil.java]: I like that real-life action implementation.

File: Mage/src/main/java/mage/constants/SubType.java [hunk 3]
```diff
@@ -201,6 +201,7 @@
     KOR("Kor", SubTypeSet.CreatureType),
     KRAKEN("Kraken", SubTypeSet.CreatureType),
     // L
+    LADYOFPROPERETIQUETTE("Lady of Proper Etiquette", SubTypeSet.CreatureType), // Unglued
```
Comment (theelk801) [inline Mage/src/main/java/mage/constants/SubType.java]: don't forget `customSet=true`

**Comment by ketsuban** [comment]
I'm a little at a loss as to what to do here. I can't believe that the fine folks at Wizards want Miss Demeanour to have four creature types (Lady, of, Proper and Etiquette) but that is clearly what the Oracle text for this card seems to be implying. (They clearly dropped the ball when errataing Unglued cards by doing nothing about the old-style creature types.) Worse, the multiword creature type didn't pose a problem until I changed the name of the card, and I have no idea why.

**Comment by ketsuban** [comment]
After some discussion on Discord (with thanks to @theelk801) I've elected to leave the subtype as it is. My reasoning is as follows: Un-creature types were removed from the Comprehensive Rules with the Grand Creature Type Update, but before that "Lady of Proper Etiquette" was listed as a single creature type. MTGJSON has apparently opted to split it into individual words, possibly because the data was entered by script from Gatherer, but I don't think anyone at any stage intended for "of" to be a creature type in games allowing silver-bordered cards.

**Pull request merged**